### PR TITLE
Fix the bug in configuration of bound and public addresses

### PR DESCRIPTION
**Pull request by AverkinSergei**
Fix the bug in configuration of bound and public addresses by .config file

### CODE REVIEW

File: pjsip4net/DefaultSipUserAgent.cs [hunk 1]
```diff
@@ -75,7 +75,7 @@ public ISipUserAgent Start()
             Container.RegisterAsSingleton(localRegistry.SipTransport);
 
             var mediaApiProvider = Container.Get<IMediaApiProvider>();
-            mediaApiProvider.CreateMediaTransport(localRegistry.RtpTransport.Config);
+            mediaApiProvider.CreateMediaTransport(localRegistry.SipTransport.Config);
```
Comment (siniypin) [inline pjsip4net/DefaultSipUserAgent.cs]: I think we should rather reuse bound and public addresses from SIP transport creating a RTP transport instead (https://github.com/siniypin/pjsip4net/blob/5c2b567b4ca64c98e697586b672ad8a476e6d87d/pjsip4net/DefaultSipUserAgent.cs#L71). pjsip does not support neither TCP not TLS as RTP transports.

Comment (AverkinSergei) [inline pjsip4net/DefaultSipUserAgent.cs]: Ok, I agree, I'll fix it.

**Comment by AverkinSergei** [comment]
Waht you think about this?

File: pjsip4net/DefaultSipUserAgent.cs (hunk 1 above)
Comment (AverkinSergei) [inline pjsip4net/DefaultSipUserAgent.cs]: Check this, please.

**Pull request merged**